companion/src/modeledit/logicalswitches.h

```diff
@@ -37,12 +38,16 @@ class LogicalSwitchesPanel : public ModelPanel
     virtual void update();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this member function so that it doesn't hide an inherited non-virtual function, or make it virtual in the base class "QWidget". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1242)